site/content/en/references/kustomize/kustomization/patchesjson6902/_index.md

```diff
@@ -26,12 +26,29 @@ The content in this patch file can be either in JSON format as
 or in YAML format as
```

Comment:
Thanks for these examples! Can you please add them to the JSON version above as well? They are meant to be equivalent.

Reply:
Thank you @KnVerey , i have added the same examples for JSON version.